### DISCUSSION TITLE
Release Pipeline - Import Solution Error

### DISCUSSION THREAD

**Discussion by BP-WS** [microsoft/powerplatform-build-tools#1023] [Q&A]
I'm new to Azure dev ops and am trying to build a very simple release pipeline, where a Power Platform solution will be imported into a 'Stage' environment. When I run this pipeline, I keep getting the error:

"The service principal with id [id] for application *** does not have permission to access the path [path]". 

I am using a service connection via an Azure app registration that already has Dynamics CRM user_impersonation access delegated.

Would be very grateful if any can point me in the right direction.

Thanks

**Comment by jbuenoliveira** [comment]
@BP-WS - Have you registered the service principal in the environments? Have you granted system administrator access?
